Centaurus.Domain/Global.cs

```diff
@@ -177,12 +205,23 @@ private set
         public static StateManager AppState { get; private set; }
         public static QuantumProcessorsStorage QuantumProcessor { get; private set; }
         public static AnalyticsManager AnalyticsManager { get; private set; }
+        public static PerformanceStatisticsManager PerformanceStatisticsManager { get; private set; }
         public static bool IsAlpha { get; private set; }
         public static IStorage PermanentStorage { get; private set; }
         public static BaseSettings Settings { get; private set; }
         public static StellarNetwork StellarNetwork { get; private set; }
         public static HashSet<int> AssetIds { get; private set; }
 
+
+        private static void DisposePerformanceStatisticsManager()
+        {
+            if (PerformanceStatisticsManager != null)
```

Comment:
Have you checked the performance implications of tracking current performance counters everywhere? It should be as lightweight as possible. Maybe adding async calls without awaiting them will be faster than doing performance metrics tracking in a sync fashion. Or maybe not. We should really create a separate issue for this and compare both approaches once we have time.

Reply:
`PerformanceStatisticsManager` reads `AlphaQuantumStorage.CurrentApex` once a second, and also it's subscribed to `PendingUpdatesManager.OnBatchSaved` event (it's invoked in a separate thread without awaiting).